apps/cowswap-frontend/src/api/gasPrices/index.ts

```diff
@@ -119,7 +119,7 @@ class GasFeeApi {
     return response.json()
   }
 
-  async getGasPrices(chainId: ChainId = DEFAULT_NETWORK_FOR_LISTS): Promise<GasFeeEndpointResponse> {
```

Comment:
nit: maybe is not bad to have a default network (imagine a widget in Gnosis Chain only app)?

Reply:
We still have it.
`DEFAULT_NETWORK_FOR_LISTS === ChainId.MAINNET`